requirements.txt

```diff
@@ -39,11 +43,12 @@ pyparsing==2.2.0
 python-dateutil==2.5.3
 python-env==1.0.0
 pytz==2017.2
-requests==2.13.0
+requests==2.16.5
 simplegeneric==0.8.1
 singledispatch==3.4.0.3
 six==1.10.0
 static3==0.7.0
 traitlets==4.2.1
 typing==3.6.1
+urllib3==1.21.1
 whitenoise==3.3.0
```

Comment:
I would suggest reverting back to previous requirements. Or better still, since this PR has just a small fix, you could make a new branch from staging again, add your change and push. then make a new PR. that still works.